tests/ui/parser/anon-enums-access.rs

```diff
@@ -0,0 +1,17 @@
+//fn foo(x: bool | i32) -> i32 | f64 {
```

Comment:
These tests don't test what they're titled, and this commented signature is misleading. Can you rename them?